static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
@guybedford We had to make this change because the manually imported `require` did not seem to run through `System.normalize`. Is this a bug in System?